CenterForOpenScience/osf.io
# [ENG-1702] Registration Initiator Bibliographic Status Bug

## Purpose
Fixes the issue where the bibliographic status of a draft_registration and registration initiator is defaulted to visible rather than copying the bibliographic status from the node the registration is based upon, assuming that the registration is created based off of an existing node.

## Changes
* Modifies the `post_save` function for draft registrations to create the DraftRegistrationContributor object using the node contributor visibility status, if available

## QA Notes
Through the UI or the API, QA should verify that the bibliographic status of the initiator of a draft_registration and registration should match the bibliographic status of the initiator on the existing node, for registrations based upon existing nodes. It would also be beneficial to ensure the creation process of draft registrations without existing nodes works as expected.

## Documentation
N/A

## Side Effects
N/A

## Ticket
[JIRA Ticket](https://openscience.atlassian.net/browse/ENG-1702)
